### PR TITLE
Add google dataproc

### LINKED ISSUE #5133: Google cloud services
Google Cloud has many services.
This is a total different realm than https://github.com/simple-icons/simple-icons/issues/4205
Note that Google Cloud has icon but also any one of it's services has it's own icon.

These services are highly popular.

Listing the most common (important ones):

- [x] BigQuery #9437
- [x] BigTable #9439
- [x] DataFlow #9450
- [x] DataProc #9451
- [x] Cloud Composer #9452
- [x] Pub/Sub #9466
- [x] Google Storage #9467
- [x] Cloud Spanner #9472
- [x] DataStudio #9473

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
googledataproc

**Issue:** contributes toward #5133 

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from https://cloud.google.com/icons
Color: Taken from logo `#aecbfa`

> **Note**
> The source itself has misaligned objects in the icon. I tried fixing them up

### CODE REVIEW

**Review by adamrusted — CHANGES_REQUESTED** [review]
This one's not particularly nice to work with, but I think if we're removing some of the shadows - we should probably remove [them all](https://github.com/simple-icons/simple-icons/assets/1830867/551ecde8-e1a0-4f45-9c21-c1a4b20126a5).

Could you work your magic on this one again please @PratyushJayachandran? 🙏

**Comment by PratyushJayachandran** [comment]
> Could you work your magic on this one again please @PratyushJayachandran? 🙏

The objects are misaligned in a lot of their icons. Someone made it in higher precision and then reduced it maybe. I'm not sure about this.
But yeah I'll give it another try. This one looked more symmetrical to me so I went with this approach.

**Comment by PratyushJayachandran** [comment]
Hello @adamrusted!

I had to use a lot of 'creative liberty' to get this one done as the shapes are misaligned in several places. Attaching some examples
image
image

**Comment by PratyushJayachandran** [comment]
New Preview
googledataproc

**Review by PeterShaggyNoble — DISMISSED** [review]
Did nobody in Google review any of these icons before releasing them to the public?! 🤦🏻‍♂️ 

I'm OK with the attempt to clean this one up, but we'll need another maintainer to give the OK too.

**Comment by PratyushJayachandran** [comment]
> Did nobody in Google review any of these icons before releasing them to the public?! 🤦🏻‍♂️
> 
> I'm OK with the attempt to clean this one up, but we'll need another maintainer to give the OK too.

Maybe they made it at a different dimension and then resized it. The objects may have lost their position due to precision issues.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5379,6 +5379,12 @@
             "source": "https://cloud.google.com/icons",
             "guidelines": "https://about.google/brand-resource-center"
         },
+        {
+            "title": "Google Dataproc",
+            "hex": "AECBFA",
```
[the remark this answers is not in the document]
Comment (adamrusted) [reply _data/simple-icons.json:5384]: I think we should go with the primary colour of the icon - in this case, @PratyushJayachandran was correct with `#AECBFA`.

**Review by adamrusted — APPROVED** [review]

**Pull request merged**